### ISSUE TITLE
1.7 animation sword model overlay support

### ISSUE THREAD

**Issue by nkomarn** [Nilsen84/lunar-client-agents#9]
A bit ago, Lunar released an update to their client which reworked the 1.7 animations mod and made it look closer to 1.7. This was not inherently a bad change, but by changing their code, they no longer support custom sword models (block hit overlays) from resource packs. A few packs that I know used this to make the sword position cleaner looking and more satisfying. The only way to get custom sword models to work at this time is to disable 1.7 animations for swords, which reverts to the choppy 1.8 block hitting animation.

Here's an example of a pack which used this and is now broken as of the recent(-ish) update: https://www.youtube.com/watch?v=_S_ygjoHGJs.

TL;DR: I would like to request an agent that adds support for overlaying sword position when 1.7 animations are enabled.
